tests/packages/mlx/test_mellanox-modules-compat.py

```diff
@@ -0,0 +1,19 @@
+# Requirements:
+# From --hosts parameter:
+# - host(A1): any master host of a pool, with access to XCP-ng RPM repositories and reports.xcp-ng.org.
+import logging
+from pkgfixtures import host_with_saved_yum_state
```

Comment:
Must be imported in `conftest.py` for the package (and it's better if the package contains an empty `__init__.py` file to indicate it is a package) for the workaround to work correctly, IIUC. (cc @ydirson)

Reply:
yes, `__init__.py` is necessary too, or the directory is not even a package and the workaround cannot help, as we saw recently